pkg/minishift/openshift/oc.go

```diff
@@ -17,17 +17,29 @@ limitations under the License.
 package openshift
 
 import (
+	"encoding/json"
+	"errors"
 	"fmt"
-	"strings"
 
 	"github.com/minishift/minishift/pkg/minikube/constants"
 	instanceState "github.com/minishift/minishift/pkg/minishift/config"
 	"github.com/minishift/minishift/pkg/util"
+	routeApi "github.com/openshift/origin/pkg/route/api"
+	"path/filepath"
+	"strings"
 )
 
 // runner executes commands on the host
 var runner util.Runner = &util.RealRunner{}
 
+var (
+	systemKubeConfigPath string
+)
+
+func init() {
+	systemKubeConfigPath = filepath.Join(constants.Minipath, "machines", constants.MachineName+"_kubeconfig")
```

Comment:
Currently, I am using format for storing the system:admin kube entries:
```
MINISHIFT_HOME/machines/MACHINE_NAME_kubeconfig
```

Suggestions are welcome if not appropriate.